xml/System.Security.Cryptography/AesCryptoServiceProvider.xml

```diff
@@ -546,9 +556,22 @@
         <ReturnType>System.Byte[]</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets or sets the initialization vector (<see cref="P:System.Security.Cryptography.SymmetricAlgorithm.IV" />) for the symmetric algorithm.</summary>
+        <value>The initialization vector.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ The <xref:System.Security.Cryptography.SymmetricAlgorithm.IV%2A> property is automatically set to a new random value whenever you create a new instance of one of the <xref:System.Security.Cryptography.SymmetricAlgorithm> classes or when you manually call the <xref:System.Security.Cryptography.SymmetricAlgorithm.GenerateIV%2A> method.  The size of the <xref:System.Security.Cryptography.SymmetricAlgorithm.IV%2A> property must be the same as the <xref:System.Security.Cryptography.SymmetricAlgorithm.BlockSize%2A> property divided by 8.  
```

Comment:
Should this really be pointing to SymmetricAlgorithm properties instead of talking about this specific class?

Reply:
The property does come from SymmetricAlgorithm.  Mainly I was copying data from the derived type because this is just "override without changing documentation", so almost everything was copy/paste with no substitution or thinking.

Aka my missing `<inheritdocs/>` feature :)